### DISCUSSION TITLE
RKNN Module not found

### DISCUSSION THREAD

**Discussion by samarth1232** [community/community#130980] [Programming Help]
I am trying to run a few ML models on an RK3588 using the RKNN toolkit, but I'm encountering an issue. Despite installing both the rknn-toolkit2 and rknn-toolkit-lite, I get the following error when I try to import the RKNN module in Python: ModuleNotFoundError: No module named 'rknn.api.rknn_log'. I've verified the installation and checked the directory structure, but the issue persists. Can anyone help me resolve this problem?

**Comment by caferyukseloglu** [comment]
Hey! The error usually happens because you're importing the wrong package on the board.

**On your RK3588, don't use:**
```python
from rknn.api import RKNN  # This is for x86 PC only
```

**Use this instead:**
```python
from rknnlite.api import RKNNLite
```

`rknn-toolkit2` is meant for your PC (model conversion), while `rknn-toolkit-lite2` is what actually runs on the board.

Also make sure:
- You grabbed the latest packages from the **airockchip** repo (the old rockchip-linux one is deprecated)
- Your Python version matches the `.whl` file you installed (e.g., cp310 = Python 3.10)
- The NPU driver is loaded — run `dmesg | grep rknpu` to check

A clean reinstall in a fresh virtual env usually sorts it out.
